docs/references/dimensions.mdx

```diff
@@ -185,7 +188,65 @@ and it will stay on multiple lines
+
+To help you build your format expression, we recommend using https://customformats.com/.
+
+```yaml
+metrics:
+  total_us_revenue:
+    type: sum
+    description: 'Total revenue in USD, with two decimal places, compacted to thousands'
+    format: '$#,##0.00," K"' # 505,430 will appear as '$505.43 K'
+  percent_of_total_global_revenue:
+    type: number
+    description: 'Percent of total global revenue coming from US revenue.'
+    sql: ${total_us_revenue} / ${total_global_revenue}
+    format: '0.00%' # 0.67895243 will appear as '67.89%
+```
+
+<details>
+  <summary>(Legacy) format, round, and compact options</summary>
+
+Spreadsheet-style format expressions are the recommended way of adding formatting to your metrics in Lightdash. There are legacy formatting options, listed below, which are less flexible than the spreadsheet-style formatting. 
+
+:::info
+
+If you use both legacy and spreadsheet-style formatting options for a single dimension, Lightdash will ignore the legacy `format`, `compact`, and `round` options and only apply the spreadsheet-style formatting expression.
+
+:::
+
+#### Format (legacy)
+
+```yaml
+version: 2
+
+models:
+  - name: sales_stats
+    columns:
+      - name: revenue
+        description: 'Total estimated revenue in GBP based on forecasting done by the finance team.'
+        meta:
+          dimension:
+            format: 'gbp'
+```
+
+These are the options:
+
+| Option  | Description                                                                         | Raw value  | Displayed value |
+| ------- | ----------------------------------------------------------------------------------- | ---------- | --------------- |
+| km      | Adds the suffix `km` to your value                                                  | 10         | 10 km           |
+| mi      | Adds the suffix `mile` to your value                                                | 10         | 10 mi           |
+| usd     | Adds the `$` symbol to your number value                                            | 10         | $10.00          |
+| gbp     | Adds the `£` symbol to your number value                                            | 10         | £10.00          |
+| eur     | Adds the `€` symbol to your number value                                            | 10         | €10.00          |
+| jpy     | Adds the `¥` symbol to your number value                                            | 10         | ¥10             |
+| percent | Adds the `%` symbol and multiplies your value by 100                                | 0.1        | %10             |
+| id      | Removes commas and spaces from number or string types so that they appear like IDs. | 12,389,572 | 12389572        |
+
+#### Compact (legacy)
 
 You can compact values in your YAML. For example, if I wanted all of my revenue values to be shown in thousands (e.g. `1,500` appears as `1.50K`), then I would write something like this in my .yml:
```

Comment:
perhaps , on this legacy sections, suggest an alternative for this using the new format. This is a common practice when we "deperecate" a function in code